flexmeasures/api/common/schemas/sensor_data.py

```diff
@@ -13,6 +15,42 @@
 from flexmeasures.data.schemas.times import AwareDateTimeField, DurationField
 
 
+def disambiguate_deserialization(data, _):
+    """Allows both a single float and a list of floats. Always returns a list of floats.
+
+    Meant to improve user experience by not needing to make a list out of a single item, such that:
+
+        {
+            "values": [3.7]
+        }
+
+    can be written as:
+
+        {
+            "values": 3.7
+        }
+
+    Either will be deserialized to [3.7].
+
+    Note that serialization always results in a list of floats.
+    This ensures that we are not requiring the same flexibility from users who are retrieving data.
+    """
+    if isinstance(data, list):
+        return fields.List(fields.Float)
+    else:
+        return SingleValueField
```

Comment:
From the PolyField documentation, I thought that we'd need to instantiate: `SingleValueField()`?